code/modules/reagents/reagents/other.dm

```diff
@@ -522,7 +522,7 @@
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
 		var/obj/item/organ/internal/heart/heart = H.random_organ_by_process(OP_HEART)
-		if(heart)
```

Comment:
This seems like a bit of an exploit, to have a robotic heart to avoid any consequences of resuscitator dosage.

Reply:
Do we have a robotic heart?
Robotic heart must have downside, that it can be totally broken with one shot of EMP.

Reply:
Eh, robotics being so easy to damage by EMP is offset by the ease to repair them.

Only needs a screwdriver

Reply:
But it can kill you in one shot.
What the point than in a robotic heart, if you can be easily killed like that?